client/python/lib/ovmsclient/tfs_compat/grpc/tensors.py

```diff
@@ -61,6 +63,39 @@ def make_tensor_proto(values, dtype=None, shape=None):
 
     raise NotImplementedError
 
+def as_numpy_dtype(tensor_dtype):
+    if tensor_dtype == 0:
+        raise ValueError("Tensor data type invalid")
+    elif tensor_dtype == 1:
+        return np.float32
+    elif tensor_dtype == 2:
+        return np.float64
+    elif tensor_dtype == 3:
+        return np.int32
+    elif tensor_dtype == 4:
+        return np.uint8
+    elif tensor_dtype == 5:
+        return np.int16
+    elif tensor_dtype == 6:
+        return np.int8
+    elif tensor_dtype == 7:
+        return np.str
+    elif tensor_dtype == 8:
+        return np.complex64
+    elif tensor_dtype == 9:
+        return np.int64
+    elif tensor_dtype == 10:
+        return np.bool
+    elif tensor_dtype == 17:
+        return np.uint16
+    elif tensor_dtype == 22:
+        return np.uint32
+    elif tensor_dtype == 23:
+        return np.uint64
```

Comment:
Can you create enum with the names of tf types so we can have named consts here instead of numbers?